scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -2,6 +2,38 @@
 {% from "utils.html" import comments_table %}
 {% from "variants/indicators.html" import pin_indicator, causative_badge, comments_badge, evaluations_badge, research_assessments_badge, clinical_assessments_badge, group_assessments_badge %}
 
+{% macro filter_script_main(cytobands) %}
```

Comment:
Nice with the reusable macro! 🥇

Reply:
But apparently it broke the populateCytobands rerun for some reason. It tested ok before. lll have a look!

Reply:
Yeah, looks like there is something odd with the argument passing via the macro. Perhaps that is why it wasn't before. :grin: Ill give it a few more minutes..

Reply:
Feel free to open a PR to switch to b5 🤣

Reply:
There is one also for b4: https://mdbootstrap.com/docs/b4/jquery/forms/multiselect/

Check the example under "Multiselect with Bootstrap styles"

Reply:
Interesting, though that particular example is bugged (note two Select All - the deselected and the selectable).

Reply:
On the macro part, that was actually ok; I was not paying attention to the priority order when passing a function with an argument to addEventListener. The function actually got executed on load and the return value passed to addEventListener. 🙄